examples/stm32/xively_demo_stm32f4xx_nucleo_eth/Src/xively_client.c

```diff
@@ -36,9 +36,9 @@
  *  Device Specific
  *  Credential Information
  */
-#define XI_ACCOUNT_ID "4d3c7986-8d53-4cf8-903e-7fd30ff63be1"
-#define XI_DEVICE_ID "6ed0ab44-d7e6-47fa-a3de-6d3441e5c577"
-#define XI_PASSWORD "diYY01MoMWFI3fCGTr7gZamtfFIMmedWrm+gkHGtLHc="
+#define XI_ACCOUNT_ID "Xively Account ID"
+#define XI_DEVICE_ID "Xively Device ID"
+#define XI_DEVICE_PASS "Xively Device Password"
 

```

Comment:
I've noticed this when getting Device Password and Account Id from the device details page, that Device ID is first and AccountID is second on that page.  It's a bit confusing.    

I'm not sure if we should change this in source or not, though.  Just bringing it up to see what others think.